Modules/Core/Common/include/itkCellInterface.h

```diff
@@ -173,7 +173,7 @@ class ITK_TEMPLATE_EXPORT CellInterface
       return smartPtr;
     }
 
-    /** Run-time type information (and related methods).   */
+    /** Defines `const char * GetNameOfClass() const override`, returning the name of this class (or class template). */
```

Comment:
Of course, we can also shorten this to something like:
```cpp
  /** Defines `const char * GetNameOfClass() const override`. */
```
Because the same line of comment will be there in 1000+ places, maybe it's better to keep it short. For more details, people may just check the macro definitions themselves! What do you think?

Reply:
Agreed.